doc/man1/s_server.pod

```diff
@@ -166,23 +212,58 @@ Use IPv4 only.
 
 Use IPv6 only.
 
-=item B<-context id>
+=item B<-unlink>
+
+For -unix, unlink existing socket first.
+
+=item B<-context val>
 
 Sets the SSL context id. It can be given any string value. If this option
 is not present a default value will be used.
 
-=item B<-cert certname>
+=item B<-verify int>, B<-Verify int>
```

Comment:
Wow, we really have a case-dependent long-form argument like that.  Not that we could change it until 1.2, of course...